cmake/Modules/FindRTIConnextDDS.cmake

```diff
@@ -345,6 +345,14 @@
 #   variable. Example:
 #       cmake -DCONNEXT_LIBS_BUILD_TYPE=Release
 #
+#   Take into account that forcing Release or Debug libraries could result in
+#   errors on Windows when building the opposite build type (e.g.:
+#   using ``-DCONNEXT_LIBS_BUILD_TYPE=Release`` and building with
+#   ``cmake --build . --config Debug``). Also, if an executable ends with the
+#   Debug and Release libraries of Connext, some problems like double-freeing
+#   memmory could occur (once for the Debug symbol and other for the Release
+#   one).
+#
```

Comment:
What if you replace, "Also, if and executable ..." entirely with just this:

You will end up with an executable with both Debug and Release libraries, resulting in problems like double-freeing memory (once for the Debug symbol and again for the Release symbol).

Reply:
Luis and I met over Slack and came up with better wording